depends/windows/angle/CMakeLists.txt

```diff
@@ -0,0 +1,372 @@
+
+if(CMAKE_SIZEOF_VOID_P EQUAL 4)
+  add_definitions(-DANGLE_IS_32_BIT_CPU=1)
+elseif(CMAKE_SIZEOF_VOID_P EQUAL 8)
+  add_definitions(-DANGLE_IS_64_BIT_CPU)
+else()
+  message(FATAL_ERROR "Unknown current CPU bit size: ${CMAKE_SIZEOF_VOID_P}")
+endif()
+
+if(WIN32 AND NOT WINDOWS_STORE)
+    set(WINDOWS_DESKTOP 1)
+else()
+    set(WINDOWS_DESKTOP 0)
+endif()
+if (WINDOWS_DESKTOP OR WINDOWS_STORE)
+    set(IS_WIN 1)
+    set(WINDOWS_ANY 1)
+else()
+    set(WINDOWS_ANY 0)
+endif()
+
+if(UNIX AND NOT APPLE)
+    set(LINUX 1)
+else()
+    set(LINUX 0)
+endif()
+
+# check win options
+if(WINDOWS_ANY)
+  if(WINDOWS_STORE)
+    if (NOT ANGLE_ENABLE_D3D11 AND NOT ANGLE_ENABLE_NULL)
+      message(ERROR "No any supported renderer enabled (D3D11 or NULL)")
+    endif()
+  else()
+    if (NOT ANGLE_ENABLE_D3D11 AND NOT ANGLE_ENABLE_D3D9 AND NOT ANGLE_ENABLE_WGL AND NOT ANGLE_ENABLE_NULL)
+      message(ERROR "No any supported renderer enabled (D3D11, D3D9, OpenGL or NULL)")
+    endif()
+  endif()
+endif()
+
+if(CMAKE_BUILD_TYPE MATCHES DEBUG)
+  set(IS_DEBUG 1)
+else()
+  set(IS_DEBUG 0)
+endif()
+
+if(WINDOWS_ANY)
+  add_compile_options(/d2guard4 /Wv:18 /guard:cf)
+else()
+  add_compile_options(-std=c++17 -fPIC)
+endif()
+
+if(APPLE)
+  set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -framework Cocoa -framework OpenGL -framework IOKit -framework CoreFoundation -framework IOSurface -framework QuartzCore -framework CoreGraphics")
+endif()
+
+set(CMAKE_MODULE_LINKER_FLAGS "${CMAKE_MODULE_LINKER_FLAGS}")
+set(CMAKE_STATIC_LIBRARY_PREFIX "")
+set(CMAKE_DEBUG_POSTFIX "_dbg")
+
+# if BUILD_SHARED_LIBS is undefined comes it to a cmake build error, set it as default to 0.
+if(NOT DEFINED BUILD_SHARED_LIBS)
+  set(BUILD_SHARED_LIBS 0)
+endif()
+
+if(NOT PACKAGE_NAME)
+  set(PACKAGE_NAME "unofficial")
+endif()
+
+add_definitions(
+  -D_CRT_SECURE_NO_DEPRECATE
+  -D_SCL_SECURE_NO_WARNINGS
+  -D_HAS_EXCEPTIONS=0
+  -DNOMINMAX
+  -DANGLE_STANDALONE_BUILD)
+if(IS_DEBUG)
+  add_definitions(-DANGLE_ENABLE_DEBUG_ANNOTATIONS)
+endif()
+if(DCHECK_ALWAYS_ON)
+  add_definitions(-DANGLE_ENABLE_RELEASE_ASSERTS)
+endif()
+
+# Shared library function visibility
+if(IS_WIN)
+  set(ANGLE_GL_VISIBILITY_CONFIG -DGL_APICALL=
+                                 -DGL_API=)
+else()
+  set(ANGLE_GL_VISIBILITY_CONFIG -DGL_APICALL=__attribute__((visibility(\"default\")))
+                                 -DGL_API=__attribute__((visibility(\"default\"))))
+endif()
+
+# This config controls export definitions on ANGLE API calls.
+set(ANGLE_STATIC_DEFINES -DANGLE_EXPORT=
+                         -DANGLE_UTIL_EXPORT=
+                         -DEGLAPI=
+                         -DGL_APICALL=
+                         -DGL_API=)
+
+set(GL_PROTOTYPES -DGL_GLES_PROTOTYPES=1
+                  -DEGL_EGL_PROTOTYPES=1
+                  -DGL_GLEXT_PROTOTYPES
+                  -DEGL_EGLEXT_PROTOTYPES
+                  -DHAS_GLES=3
```

Comment:
I think what `HAS_GLES` shouldn't be a part of angle interface.

Reply:
Moved back to FindGLES.cmake.